clients/datasetapi.go

```diff
@@ -0,0 +1,17 @@
+package clients
+
+import (
+	"context"
+
+	datasetclient "github.com/ONSdigital/dp-api-clients-go/v2/dataset"
+)
+
+//go:generate moq -out mock/datasetapi.go -pkg mock . DatasetClient
+
+// DatasetAPIClient defines the zebedee client
```

Comment:
comment needs updating to something like:

```suggestion
// DatasetAPIClient defines the dataset client
```

Reply:
Fixed in update 1.